modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -548,16 +548,8 @@ public class IgniteImpl implements Ignite {
         cmgLogStorageFactory =
                 SharedLogStorageFactoryUtils.create(clusterSvc.nodeName(), cmgWorkDir.raftLogPath());
 
-        RaftOptionsConfigurator cmgRaftConfigurator = options -> {
-            RaftGroupOptions raftOptions = (RaftGroupOptions) options;
-            // TODO: use interface, see https://issues.apache.org/jira/browse/IGNITE-18273
-            if (raftOptions.getLogStorageFactory() == null) {
-                raftOptions.setLogStorageFactory(cmgLogStorageFactory);
-            }
-            if (raftOptions.serverDataPath() == null) {
-                raftOptions.serverDataPath(cmgWorkDir.metaPath());
-            }
-        };
+        RaftGroupOptionsConfigurer cmgRaftConfigurator =
```

Comment:
Type says that it's configurer, but variable is a configurator. Should they agree?